app/views/prosecution_cases/_hearing_summaries.html.haml

```diff
@@ -13,7 +13,12 @@
       - hearing.hearing_days.map(&:to_date).sort.each do |hearing_day|
         %tr.govuk-table__row
           %td.govuk-table__cell
-            = link_to hearing_day.to_date.strftime('%d/%m/%Y'), hearing_path(id: hearing.id, urn: prosecution_case.prosecution_case_reference, hearing_day: hearing_day.to_date.strftime('%d/%m/%Y')), class: 'govuk-link'
+            - if hearing_day > Date.today
```

Comment:
better to use rails `Date.current` - see [docs for difference](https://api.rubyonrails.org/v5.1.7/classes/Date.html#method-c-current)

Also, not ideal to put conditional logic in the view. To handle it could be moved to a decorator. However, as part of pagination I am looking at amending this so probably
not worth it at the moment. 

on the issue of pagination I can see this is going to cause issues as well as it effectively means we have to skip over some days. For that reason I am not sure we should be doing this now at all unless this is an established feature of Common platform - what is the actual harm in linking to a future hearing anyway? Won't it just will give any available details for the hearing/day?!

Reply:
👍